client/client.go

```diff
@@ -58,6 +58,23 @@ func (c *httpClient) RequestURL(u *url.URL) (*http.Response, error) {
 	if err != nil {
 		return resp, err
 	}
+	// Check if we can bypass auth checks via X-Forwarded-For & X-Real-IP
```

Comment:
Should probably add a flag to enable/disable this, as it could greatly increase number of requests.